Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -1081,7 +1081,7 @@ void VideoConfigDiag::PopulatePostProcessingShaders()
   if (shaders.empty())
     return;
 
-  choice_ppshader->AppendString(_("(off)"));
+  choice_ppshader->AppendString(_("Off"));
```

Comment:
It's probably for making it stand out more from the other options (and there are a lot of options). We do similar things elsewhere, e.g. <Nothing> in the EXI slot config.